libs/design/src/molecules/sidebar/sidebar-viewport/sidebar-viewport.component.scss

```diff
@@ -1,67 +1,38 @@
-$sidebar-width: 250px;
+@use '../helper/variables';
 
-// Stacking Context Layers
-$daff-sidebar-z-index: 4;
-$daff-sidebar-backdrop-z-index: 3;
-$daff-sidebar-content-z-index: 2;
-$daff-sidebar-viewport-z-index: 1;
+:host {
+	overflow: hidden;
+	max-height: var(--sidebar-height);
```

Comment:
I'm not sure if this is necessary. I removed all instances of height on the sidebar components, added a height in the usage css and not seeing anything wrong.